model/user/user.go

```diff
@@ -35,9 +36,16 @@ type PubKey struct {
 }
 
 type UserSettings struct {
-	Timezone     string     `json:"timezone" bson:"timezone"`
-	NewWaterfall bool       `json:"new_waterfall" bson:"new_waterfall"`
-	GithubUser   GithubUser `json:"github_user" bson:"github_user,omitempty"`
+	Timezone      string                  `json:"timezone" bson:"timezone"`
+	NewWaterfall  bool                    `json:"new_waterfall" bson:"new_waterfall"`
+	GithubUser    GithubUser              `json:"github_user" bson:"github_user,omitempty"`
+	SlackUsername string                  `bson:"slack_username" json:"slack_username"`
+	Notifications NotificationPreferences `bson:"notifications" json:"notifications"`
+}
+
+type NotificationPreferences struct {
+	BuildBreak  event.UserSubscriptionPreference `bson:"build_break" json:"build_break"`
```

Comment:
omitempty on these as well?

Reply:
I feel like these ones shouldn't be omitempty if the notification prefs are omitempty